### ISSUE TITLE
PhyCell_Cell code is different comparing to the paper

### ISSUE THREAD

**Issue by sippeyxp** [vincent-leguen/PhyDNet#25]
In paper K is a function of h_tilde_t and E(u_t). 

In code, the implementation is 

24        combined = torch.cat([x, hidden], dim=1)  # concatenate along channel axis
25        combined_conv = self.convgate(combined)
26        K = torch.sigmoid(combined_conv)

On line 24, it should be  torch.cat([x, hidden_tilde, dim=1) ?

**Comment by lf-demo** [comment]
I also have the same question. As is described in paper, the new hidden state should be the result of eq 6 ?

**Comment by lf-demo** [comment]
I have the same question with you at first. But i find that the detail of experiment is recorded in the supplementary. Here is the link [https://openaccess.thecvf.com/content_CVPR_2020/supplemental/Le_Guen_Disentangling_Physical_Dynamics_CVPR_2020_supplemental.pdf](url)
